specification/dimensions/resourcetype.md

```diff
@@ -0,0 +1,34 @@
+# Resource Type
+
+Resource Type describes the kind of resource for which you are being charged.  A Resource Type is commonly used for scenarios like identifying cost changes in groups of similar resources and MAY include values like Virtual Machine, Data Warehouse, and Load Balancer.
+
+The ResourceType column MUST be present within billing data.  This column MUST be of type String and MUST NOT be null when a corresponding ResourceId is not null.  When a corresponding ResourceId value is null, the ResourceType column value MUST also be null.  Providers MUST use a consistent value-format and a set of values for ResourceType values within the dataset.
+
+## Column ID
+
+ResourceType
+
+## Display Name
+
+Resource Type
+
+## Description
+
+The kind of resource for which you are being charged.
+
+## Data type
+
+String
+
```

Comment:
Where are we getting these random headings from :) ? Data type is already in the content constraints below.